en/docs/assets/js/theme.js

```diff
@@ -49,3 +49,47 @@ for (var i = 0; i < dropdowns.length; i++) {
         }
     };
 };
+
+function showDiv() {
```

Comment:
Since we are adding this to theme.js would be better to use some meaningful function name than generic to avoid confusions and more readability